base/loading.jl

```diff
@@ -1578,7 +1580,7 @@ function collect_preferences!(project_toml::String, uuid::UUID)
         toml_path = joinpath(project_dir, name)
         if isfile(toml_path)
             prefs = parsed_toml(toml_path)
-            push!(dicts, get(prefs, pkg_name, Dict{String,Any}())::Dict{String,Any})
+            push!(dicts, get(Dict{String, Any}, prefs, pkg_name)::Dict{String,Any})
```

Comment:
I didn't know this form of `get()` existed!

Reply:
Isn't it `get!` and not `get`?

Reply:
There are two versions, `get!` inserts it if it doesn't exist, `get` just returns the evaluation of the function if it doesn't exist.